app/main.py

```diff
@@ -1,6 +1,16 @@
 from typing import Callable
+from functools import wraps
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    unique_values = {}
+
+    @wraps(func)
+    def inner(*args) -> Callable:
```

Comment:
`inner` can return not only `Callable`, it can return `Any`. Indicate this in the return type annotation

Reply:
Ok, got it.